sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsWithDownleveledStorage.ts

```diff
@@ -0,0 +1,88 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to use the EventHubConsumerClient to process events from all partitions
+  of a consumer group in an Event Hubs instance, as well as checkpointing along the way.
```

Comment:
The term "checkpointing" is probably a well understood comment, but someone totally new to the service would not immediately know what "checkpointing" is. However, if you describe it as "...as well as persisting your events offsets along the way." everyone would understand what we are talking about. I doubt that even experienced users coming from the Kafka ecosystem would immediately know with 100% certainty what checkpointing is. I know it is used elsewhere in our documentation and once you are understanding it is a common word with known semantics. I am saying, let's improve the experience so that anyone (new or experienced) understand the very first time what meaning we are conveying. "Checkpointing" is not a well-known phrase in the industry like "cache", "OOP", etc.

Reply:
In this specific case though, they would be looking at a sample for `eventhubs-checkpointstore-blob`. The word `checkpoint` is in the name of the package, and the README describes what checkpointing is as well. I can add some verbiage around persistence but don't want to remove the word checkpointing altogether.